firefox/scripts/firefox-esr-migration

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Migrate Firefox ESR profile to Firefox
+# Set default browser to Firefox if needed
+# Update desktop file in Phosh favorites if needed
+
+export DISPLAY=:0
+
+if [[ ! -f ~/.mozilla/firefox/installs.ini || ! -f ~/.mozilla/firefox/profiles.ini ]]
+then
+	echo "Can't find any Firefox ESR profile"
+	exit 1
```

Comment:
I think we can `exit 0` here to also properly handle new installs (where no firefox profile is present indeed)